java/data/trial_spawner.mcdoc

```diff
@@ -0,0 +1,29 @@
+use ::java::server::world::block::spawner::SpawnPotential
```

Comment:
This is a bit inelegant, but moving `SpawnPotential` here is as inelegant as it's also used by Monster Spawners